### ISSUE TITLE
Github Repository cannot be added.

### ISSUE THREAD

**Issue by jeremyje** [Shippable/support#5085]
Cannot add private GitHub repository,

`getValidCollaboratorByProjectId for projectId 5e7599c6d60a4500064d3018 returned an error: Failed to get valid collaborator account for project :id 5e7599c6d60a4500064d3018`

Quite a few of my repositories have this issue. I notice that it does not have any webhooks installed.

**Comment by a-murphy** [comment]
It looks like your account has only allowed `write:repo_hook` when `admin:repo_hook` would be expected.  Could you try revoking access on GitHub (https://help.github.com/en/github/authenticating-to-github/reviewing-your-authorized-applications-oauth), logging out of Shippable, and then logging back in to reset the scopes?

**Comment by jeremyje** [comment]
Yeah I started messing with the scopes. I think I deleted the old token. I found the shippable docs that pointed me to the scopes necessary. It was very hard to find. I cannot find it in my history. :/

After doing full access to scopes it was still broken I had 2 github integrations (an additional account for an org) and even after making sure they manage separate accounts it still was broken. Deleting the extra github integration fixed the problem.
